pkg/arvo/app/chat-hook.hoon

```diff
@@ -263,7 +341,7 @@
     ?.  =(u.shp src.bol)
       [~ this]
     :_  this(synced (~(del by synced) path.diff))
-    :-  (chat-poke diff)
+    :-  (chat-poke [%delete path.diff])
```

Comment:
Isn't this equivalent to the previous code (passing in `diff` as-is)?

Same for the `%message` case below. What am I missing?

Reply:
Type nesting changes for `chat-base`, `chat-update`, and `chat-action`.